### ISSUE TITLE
[rebber]\begin{array]{l r} breaks the latex

### ISSUE THREAD

**Issue by artragis** [zestedesavoir/zmarkdown#471]
When we parse some math operation into latex, the \\ may be removed

# input

```
$$
\begin{array}{l r}
  & 45 \\
+ & 218 \\
+ & 800 \\
\hline
= & 1063
\end{array}
$$
```

# Actual result

```
\[ begin{array}{l r}
  & 45   
+ & 218   
+ & 800  \
\hline
= & 1063
\end{array} \]
```

# Expected result

```
\[ 
\begin{array}{l r}
  & 45  \\
+ & 218 \\  
+ & 800  \\
\hline
= & 1063
\end{array} \]
```

**Comment by gilcot** [comment]
Also check if it's the case with `\left` and `\right` 

I think solving this issue should solve #137 (anyway related)

**Comment by artragis** [comment]
it is possibly related,
we can also note that the `\\` to say "line return" are not well parsed.

I also made some tests with other environments and got good parsing for 

```
\begin{matrix}
   a & b \\
   c & d
\end{matrix}
```

this bug is really strange

**Comment by StaloneLab** [comment]
Fixed in version 11.1.0, thanks for the report @gilcot
